core/src/com/unciv/ui/cityscreen/ConstructionsTable.kt

```diff
@@ -261,7 +261,7 @@ class ConstructionsTable(val cityScreen: CityScreen) : Table(CameraStageBaseScre
                 && UncivGame.Current.worldScreen.isPlayersTurn
                 && !city.isPuppet) {
             val constructionGoldCost = construction.getGoldCost(city.civInfo)
-            purchaseConstructionButton = TextButton("Buy for [$constructionGoldCost] gold".tr(), CameraStageBaseScreen.skin)
+            val purchaseConstructionButton = TextButton("Buy for [$constructionGoldCost] gold".tr(), CameraStageBaseScreen.skin)
```

Comment:
Maybe strip this change out of the PR: what you see is an artifact introduced by the merge. It will be fixed in the dedicated PR #1648.

Reply:
Thank you, will remove that.

Reply:
I need to keep this change for a while, in order to ensure that the tests are green, not failing due to compilation error. I assume it would be eliminated during rebase when the PR be applied.